vspace/pom.xml

```diff
@@ -23,10 +23,10 @@
         <log.level>debug</log.level>
 
         <db.driver>com.mysql.jdbc.Driver</db.driver>
-        <db.url>jdbc:mysql://localhost:3306/vspace?serverTimezone=UTC</db.url>
-        <db.user>vspace</db.user>
-        <db.password>vspace</db.password>
-        <uploaded.files.path></uploaded.files.path>
+        <db.url>jdbc:mysql://localhost:3307/vspace?serverTimezone=UTC</db.url>
+        <db.user>root</db.user>
+        <db.password>root</db.password>
+        <uploaded.files.path>/Users/pradeep/DeveloperAtDiging/storage-docker</uploaded.files.path>
```

Comment:
don't change this values. these should all be set via settins.xml not. changed in the pom.